flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+    "only-arches": [
+        "x86_64",
+        "aarch64"
```

Comment:
1. you don't need the file with that setup. no other architecture is support by flathub
2. you will need to revert to providing pre-built binaries since Flutter is incapable of working in that setup. that probably mean not supporting aarch64.